src/FSharp.AWS.DynamoDB/Expression/ExprCommon.fs

```diff
@@ -183,41 +182,46 @@ type QuotedAttribute =
         let tryGetPropInfo (properties: PropertyMetadata[]) isFinalProp (p: PropertyInfo) =
             match properties |> Array.tryFind (fun rp -> rp.PropertyInfo = p) with
             | None -> None
-            | Some rp when rp.Pickler.PicklerType = PicklerType.Serialized && not isFinalProp ->
+            | Some rp when
+                rp.Pickler.PicklerType = PicklerType.Serialized
```

Comment:
ew, but at least && is start of line
but would put when at start of line if adopting this (can it be turned off?)

Reply:
I don’t think there’s a way to configure `when` at the start of the line. I’m not even sure which max width is being invoked here tbh

Reply:
yeah, not to worry, minor compared to other things